src/VirtualMonoRepo/source-mappings.json

```diff
@@ -36,29 +36,14 @@
     // The default filters do not apply to submodules
     // Only filters which start with submodule's path are applied when syncing submodules
     "defaults": {
-        "defaultRef": "main",
-        "exclude": [
-            "**/*.dll",
-            "**/*.Dll",
-            "**/*.exe",
-            "**/*.mdb",
-            "**/*.nupkg",
-            "**/*.pdb",
-            "**/*.tgz",
-            "**/*.zip"
-        ]
+        "defaultRef": "main"
     },
 
     // Each of these mappings has a corresponding folder in the src/ directory
     "mappings": [
         {
             "name": "arcade",
-            "defaultRemote": "https://github.com/dotnet/arcade",
-            "exclude": [
-                "src/Microsoft.DotNet.SignTool.Tests/*.mpack",
-                "src/Microsoft.DotNet.SignTool.Tests/*.msi",
-                "src/Microsoft.DotNet.SignTool.Tests/*.vsix"
-            ]
```

Comment:
Why are these excluded? Do we need a tracking issue to bring these in differently, i.e. via a test package?

Reply:
These are not excluded anymore after this PR.

Reply:
Oh disregard, I read the diff incorrectly. Thanks.